README.md

```diff
@@ -40,5 +40,10 @@ spec:
       requests:
         memory: 100Mi
         cpu: 20m
-
 ```
+
+# Common features
+
+This is a list of supported features provided by libraries.
+1. CradleMaxMessageBatchSize - this option defines the maximum message batch size in bytes.
+   Please see more details about this feature via [link](https://github.com/th2-net/th2-common-j#configuration-formats)
```

Comment:
Do we need this information here? It is described in th2-common-j and I don't see any reason to duplicate it here. The user won't be able to change it anyway if he or she modifies the mstore configuration

Reply:
I think we should provide the list of supported features in a project which builds a docker image. We can use another way for that in the future.